src/main/kotlin/no/skatteetaten/aurora/mokey/model/ApplicationData.kt

```diff
@@ -31,22 +32,13 @@ data class ManagementEndpointError(
         val rootCause: String? = null
 )
 
-data class ManagementData constructor(
+
+data class ManagementData(
```

Comment:
Vi kan godt lage links som en Either her også. Det er bare litt vanskelig med måten vi lager EndpointFactory på akkurat nå synes jeg. 

@jarlehansen du pratet noe om noe HATEOAS her for linking kan du utdype det?

Reply:
Ja, jeg tror management apiet tilbyr hateoas hal struktur. Grunnen til at det ser sånn ut for meg er "_links".

spring-hateoas har en klasse som heter Resource:
https://docs.spring.io/spring-hateoas/docs/current/api/org/springframework/hateoas/Resource.html

Det er også en for liste-ressurser som heter Resources:
https://docs.spring.io/spring-hateoas/docs/current/api/org/springframework/hateoas/Resources.html

Denne kan brukes som vanlig inn i resttemplate og da får man parsing av linker automatisk. Så det use caset jeg ser for meg er at vi kan bruke spring-hateoas til å erstatte mye av den manuelle parsingen av dette nå. Også tror jeg resultatet burde puttes inn i våre egne DTOer etterpå, slik at vi ikke returnerer noe som er spring-hateoas spesifikt.